plenum/common/ledger_manager.py

```diff
@@ -271,7 +271,18 @@ def processLedgerStatus(self, status: LedgerStatus, frm: str):
                                  "ledger status {} from client {}"
                                  .format(self, status, frm))
                     return
-
+                try:
+                    getattr(status, f.PROTOCOL_VERSION.nm)
```

Comment:
Do we need it in both here and `_invalid_client_ledger_status_handling`?

Reply:
ok

Reply:
fixed